pom.xml

```diff
@@ -25,17 +25,18 @@
       src/main/java/org/folio/entlinks/model/**
     </sonar.exclusions>
 
+    <spring-boot.version>3.2.0</spring-boot.version>
```

Comment:
Do we really need this?

Reply:
apparently, no, we don't, thanks removed it